pigx-bsseq.in

```diff
@@ -110,6 +110,15 @@ parser.add_argument('--unlock', dest='unlock', action='store_true',
                     help="""\
 Recover after a snakemake crash.""")
 
+parser.add_argument('--verbose', dest='verbose', action='store_true',
+                    help="""\
+Print supp info on job execution.""")
+
+parser.add_argument('--printshellcmds', dest='printshellcmds', action='store_true',
+                    help="""\
+Print commands being executed by snakemake.""")
```

Comment:
What do you think about making this part of "--verbose" instead?

Reply:
they are different options.  `--verbose` tells you stuff about where the job got sent to, etc. `--printshellcmnds` prints the actual shell commands. I thought it made sense to make the arguments correspond as directly as possible with the snakemake commands

Reply:
I have exactly the opposite impulse: I try to liberate the user interface from the reality of snakemake.

Why would I want to print shell commands if not for debugging?

Reply:
ok, I'm not really understanding what the objection is to this, and if we were to merge this branch now it would really help me solve some other problems. @rekado , Are you ok with the latest commit?